Specification Registry/Attributes/name.json

```diff
@@ -0,0 +1,11 @@
+{
+    "fieldInternalName": "name",
```

Comment:
I imagine we only need a single `name` field instead of `name` and the other partial name fields

Reply:
Got it. By partial names I guess you are referring to the first, last and middle name fields right ? Will remove them and keep only the ```name``` field.